messages.py

```diff
@@ -68,120 +66,131 @@ def handle(self, response):
     def message_handler(self, data):
         """Handle chat message packets from Beam."""
 
-        parsed = ''.join([obj["text"] for obj in data["message"]["message"]])
-
-        self.logger.info("{bot}{me}[{user}] {message}".format(
-            bot='$ ' if data["user_name"] == self.config["auth"]["username"]
-                else '',
-            me='*' if data["message"]["meta"].get("me") else '',
-            user=data["user_name"] + " > " + self.config["auth"]["username"]
-                if data["message"]["meta"].get("whisper")
-                else data["user_name"],
-            message=parsed)
-        )
-
-        user = session.query(User).filter_by(id=data["user_id"]).first()
-        if user is not None:
-            user.messages += 1
-            session.commit()
+        try:
```

Comment:
This `try/except` should be unnecessary, thanks to Beam fixing this weird `text`/`data` confusion.